### PR TITLE
rare double latched toolboxes and very rare triple latched toolboxes

### PR DESCRIPTION
**Pull request by Militaires**
:cl: Basilman
add: some toolboxes, very rarely, have more than one latch
/:cl:

for joan fuck itiwnadoaiwdoidio

### CODE REVIEW

**Comment by Militaires** [comment]
conflicts resolved but travis is garbage kill me

**Comment by JJRcop** [comment]
Sorry, what was that?

**Comment by ShadowDeath6** [comment]
>itiwnadoaiwdoi*dio*

joj reference

File: code/game/objects/items/weapons/storage/toolbox.dm [hunk 1]
```diff
@@ -377,4 +392,4 @@
 				force = 40
 				spawn(400)
 					if(src)
-						warning_messages += "starving"
+						warning_messages += "starving"
```
Comment (lordpidey) [inline code/game/objects/items/weapons/storage/toolbox.dm]: Why did you take out the linebreak at end of file?

**Comment by kevinz000** [comment]
where's the needs pictures tag?

File: code/game/objects/items/weapons/storage/toolbox.dm [hunk 2]
```diff
@@ -151,7 +162,7 @@
 	item_state = "artistic_toolbox"
 	max_combined_w_class = 20
 	storage_slots = 10
-	w_class = WEIGHT_CLASS_GIGANTIC //Holds more than a regular toolbox!
+	w_class = 5 //Holds more than a regular toolbox!
```
Comment (optimumtact) [inline code/game/objects/items/weapons/storage/toolbox.dm]: What do you think you are doing

**Comment by Militaires** [comment]
@optimumtact accidential revert will fix

File: code/game/objects/items/weapons/storage/toolbox.dm [hunk 3]
```diff
@@ -13,6 +13,17 @@
 	origin_tech = "combat=1;engineering=1"
 	attack_verb = list("robusted")
 	hitsound = 'sound/weapons/smash.ogg'
+	var/image/hinges = "single_hinge"
```
Comment (AnturK) [inline code/game/objects/items/weapons/storage/toolbox.dm]: This is not image and you never actually set it to image so don't type it as such.

File: code/game/objects/items/weapons/storage/toolbox.dm [hunk 4]
```diff
@@ -13,6 +13,17 @@
...
+	var/old = FALSE
+
+/obj/item/weapon/storage/toolbox/New()
+	..()
+	if(!old)
+		if(prob(10))
+			hinges = "double_hinge"
+		else if(prob(1))
+			hinges = "triple_hinge"
+	add_overlay(image('icons/obj/storage.dmi', "[hinges]"))
```
Comment (AnturK) [inline code/game/objects/items/weapons/storage/toolbox.dm]: Move this to separate update_icon then call that.

**Pull request merged**